WalletWasabi.Gui/Controls/WalletExplorer/SendControlView.xaml

```diff
@@ -52,8 +51,8 @@
               Foreground="{Binding AmountText, Converter={StaticResource AmountForegroundConverter}}"
               IsReadOnly="{Binding IsMax}"
               Text="{Binding AmountText, Mode=TwoWay}"
-              ToolTip.Tip="{Binding UsdExchangeRate, Converter={StaticResource UsdExchangeRateAmountToolTipConverter}}"
-              Watermark="{Binding AmountWatermarkText}"
+              ToolTip.Tip="{Binding AmountTip}"
```

Comment:
ToolTip color is green for me if I press the Max button. It should be white.

Reply:
IMO it should green (like the btc price) and it lets the user know that the Max button is clicked.

Reply:
It is hard to read for me, can you make it white?

Reply:
If it is hard to read for you then maybe we should change the Amount green color when Max button is clicked
https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Gui/Converters/AmountForegroundConverter.cs#L19

> can you make it white?

I didn't find a way to do that. @jmacato how can I change only the color of the toottip?

Reply:
How about Remove ToolTip.Tip="{Binding AmountTip}" and someting like this? 

https://github.com/molnard/WalletWasabi/blob/fd209e2312a22c32a8e0d62951001a3886c4f562/WalletWasabi.Gui/Controls/WalletExplorer/ReceiveTabView.xaml#L28

With this you could set the Forecolor or add a Textbox inside the Tooltip and set the color of it.

Reply:
I agree with @molnard here if his assumption is correct (it makes sense to me btw)

Reply:
Done.